lua/lspconfig/server_configurations/haxe_language_server.lua

```diff
@@ -1,18 +1,36 @@
 local util = require 'lspconfig.util'
 
+local function find_hxml(path)
+  return vim.fs.find(function(name)
```

Comment:
use vim.fs.root

Reply:
[`vim.fs.root`](https://neovim.io/doc/user/lua.html#vim.fs.root()) seems to return the parent directory, but I'd like to find a file. e.g. `/path/to/project/build.hxml`